templates/controllers/modals/submission/viewSubmissionMetadata.tpl

```diff
@@ -9,7 +9,7 @@
  *}
 
 <div id="viewSubmissionMetadata" class="">
-	<h3>{$publication->getLocalizedFullTitle()|strip_unsafe_html}</h3>
+	<h3>{$publication->getLocalizedFullTitle(null, 'html')|unescape:'html'}</h3>
```

Comment:
Hmm, at a glance, I'm not sure why this would come from `$publication` escaped (and thus needing an unescape). Am I missing it?

Reply:
yes right. it came from a mass select/replacement of similar format . fixed .